python/python/tests/test_async_client.py

```diff
@@ -117,7 +125,15 @@ async def test_register_client_name_and_version(self, redis_client: TRedisClient
         assert "lib-name=GlidePy" in info
         assert "lib-ver=0.1.0" in info
 
-    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize(
```

Comment:
Testing all permutations of cluster_modeXprotocol can be done with pytest like that:
```
@pytest.mark.parametrize("cluster_mode", [True, False])
@pytest.mark.parametrize("protocol",  [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
async def test()...
```
It will allow us to easily add.remove resp versions

Reply:
+ It's more readable